srcpkgs/base-files/INSTALL

```diff
@@ -2,38 +2,37 @@ make_system_dirs() {
 	#
 	# Install FHS system directories.
 	#
+	umask 022
 	for d in boot etc etc/modprobe.d etc/modules-load.d \
 		etc/skel home dev proc usr mnt opt sys media var run/lock; do
-		[ ! -d ${d} ] && install -d ${d}
+		mkdir -p ${d}
 	done
 
-	[ ! -d root ] && install -dm750 root
+	mkdir -p root && chmod 750 root
```

Comment:
But this does not do the same thing. Previously if the root already existed with permissions of (for example) 0700, it would be left alone. Now you are forcibly changing the mode regardless of whether it already exists or not.

Also FWIW busybox and toybox mkdir support the `-m` flag. I would rather just use that as it does exactly what we want here: leaves the mode alone if the directory already exists.

My comment here applies for all of the mkdir && chmod lines you added, except the last two. I guess for the tmp dirs it is desired to always set the mode to 1777. So you can leave those two lines how you have them.